src/components/Categories/Categories.jsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-unused-vars */
```

Comment:
- [ ] As mentioned by the previous reviewer kindly remove this linter disabling lines for `no-unused-vars`. I see the unused var in most of the files where you added this line:

  ```
  import ReactDOM from 'react-dom';
  ```

  You can simply remove this import from the affected files since you are not really using it in those files.

  I remove ALL occurrences of the linter disabling and the app still worked as expected without any errors. ESLint (`npx eslint .`) did not return any linter errors. Kindly verify this and make the necessary changes. Remove all eslint-disable lines from your code as suggested earlier.